src/request/views.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+from api import settings as api_settings
+from api.utils import success_response
+from rest_framework import generics
+
+from .controllers.create_request_controller import CreateRequestController
+from .models import Request
+from .serializers import RequestSerializer
+
+
+class RequestsView(generics.GenericAPIView):
+    serializer_class = RequestSerializer
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def post(self, request):
+        """Create a request."""
+        try:
+            data = json.loads(request.body)
+        except json.JSONDecodeError:
+            data = request.data
+        return CreateRequestController(request, data, self.serializer_class).process()
+
+    def get(self, request):
+        """Get all of a user's active requests."""
+        user = request.user.person
+        awaiting_approval = Request.objects.filter(
```

Comment:
this should go inside of a controller (our current style is putting endpoint logic into these controller files). Can we also change the response fields to something like `awaiting_approval` and `pending_requests`